src/main/java/service/StorageService.java

```diff
@@ -0,0 +1,13 @@
+package service;
+
+import java.util.List;
+
+public interface StorageService {

```

Comment:
Let's rename this repository service according to the entity it works with. In this case, it must be named FruiteDao.

Reply:
Do you use this interface somewhere?